sys/netpfil/pf/if_pfsync.c

```diff
@@ -1384,19 +1511,20 @@ pfsyncioctl(struct ifnet *ifp, u_long cmd, caddr_t data)
 			sifp = NULL;
 		else if ((sifp = ifunit_ref(pfsyncr.pfsyncr_syncdev)) == NULL)
 			return (EINVAL);
-
-		if (sifp != NULL && (
-		    pfsyncr.pfsyncr_syncpeer.s_addr == 0 ||
-		    pfsyncr.pfsyncr_syncpeer.s_addr ==
-		    htonl(INADDR_PFSYNC_GROUP)))
+		if (sifp != NULL && (sc->sc_sync_peer.sa.sa_family == AF_UNSPEC)) {
```

Comment:
This forces IPv4 multicast as default when IP address is not given. I'd love to see IPv6 instead.